src/dom-to-image-more.js

```diff
@@ -625,6 +673,10 @@
             return url.search(/^(data:)/) !== -1;
         }
 
+        function isDimensionMissing(value) {
```

Comment:
A new helper to determine if a `width` or `height` is not specified in `px` (or not specified at all)